test/test_graph.jl

```diff
@@ -0,0 +1,56 @@
+## Test sparse
+#  workspace()
+# using ProximalOperators
+#
+# using Base.Test
+
+srand(0)
+
+m, n = (10, 50)
+
+A = sprand(m, n, 0.3)
+Adf = full(A)
+B = [A -speye(m)]
+Bd = full(B)
+
+
+
+fiag = ProximalOperators.IndGraph(A)
+fiaf = ProximalOperators.IndAffine(B, zeros(m))
+fiagdf = ProximalOperators.IndGraph(Adf)
+
+c = randn(n) * 10 + 10
+d = A * c
+
+@test fiag(c, d) == 0.0
+@test fiagdf(c, d) == 0.0
+
+y = zeros(d)
+x = zeros(c)
+
+xy = [x ; y]
+cd = [c ; d]
+
+x_a = @view xy[1:n]
+y_a = @view xy[n+1:end]
+
+## IndAffine QR
+prox!(xy, fiaf, cd)
+@test fiaf(xy) == 0.0
+
+## SPARSE
+# test call
+prox!(x, y, fiag, c, d)
```

Comment:
I didn't get it.
I mean, I've included all cases in `test_calc` and one hardcoded case in `test_results`. And since it is done by `prox_test` and other framework functions it uses exactly `prox!(xy, f, cd, gamma)` signature.

I had manual calls in previous commits, but they are less comprehensive than `prox_test`, so I cleaned it out.